src/com/oltpbenchmark/benchmarks/tpcc/TPCCLoader.java

```diff
@@ -592,7 +603,8 @@ protected int loadOrders(Connection conn, int w_id, int districtsPerWarehouse, i
 		            if (oorder.o_carrier_id != null) {
 		                ordrPrepStmt.setInt(idx++, oorder.o_carrier_id);
 		            } else {
-		                ordrPrepStmt.setNull(idx++, Types.INTEGER);
+				ordrPrepStmt.setInt(idx++, -1);
```

Comment:
Is this Clickhouse specific? What does "-1" mean?

Reply:
-1 is just an arbitrary choice. This is supposed to replace the null value but -1 might not be a good choice.